storage/gcloud/aio/storage/storage.py

```diff
@@ -149,9 +149,11 @@ async def delete(self, bucket: str, object_name: str, *,
         return data
 
     async def download(self, bucket: str, object_name: str, *,
+                       headers: dict = {},
```

Comment:
Please have the default here be `None`. Using a mutable object as a default value can lead to some [unexpected behaviour in Python](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) and should generally be avoided.